tracer.go

```diff
@@ -0,0 +1,46 @@
+package cleisthenes
+
+import (
+	"strings"
+	"sync"
+
+	"github.com/DE-labtory/iLogger"
+)
+
+type Tracer interface {
+	Log(keyvals ...interface{})
+	Trace()
+}
+
+type DefaultTracer struct {
+	lock      sync.RWMutex
+	traceList []string
+}
+
+func NewDefaultTracer() *DefaultTracer {
+	return &DefaultTracer{
+		lock:      sync.RWMutex{},
+		traceList: make([]string, 0),
+	}
+}
+
+func (t *DefaultTracer) Log(keyvals ...interface{}) {
+	t.lock.Lock()
+	defer t.lock.Unlock()
+
+	kvs := make([]string, 0)
+	for i := 0; i < len(keyvals); i += 1 {
+		kvs = append(kvs, keyvals[i].(string))
+	}
+	trace := strings.Join(kvs, " ")
+	t.traceList = append(t.traceList, trace)
+}
+
+func (t *DefaultTracer) Trace() {
```

Comment:
We need to adjust log level!

Reply:
how about add this topic to issue? I think it is not a small problem maybe